pkg/cmd/pulumi/util.go

```diff
@@ -1165,6 +1179,39 @@ func promptUser(msg string, options []string, defaultOption string, colorization
 	return response
 }
 
+// promptUserMultiSkippable wraps over promptUserMulti making it skippable through the "yes" parameter
+// commonly being the value of the --yes flag used in each command.
+// If yes is true, defaultValue is returned without prompting.
+func promptUserMultiSkippable(yes bool, msg string, options []string, defaultOptions []string,
+	colorization colors.Colorization,
+) []string {
+	if yes {
+		return defaultOptions
+	}
+	return promptUserMulti(msg, options, defaultOptions, colorization)
+}
+
+// promptUserMulti prompts the user for a value with a list of options, allowing to select none or multiple options.
+// defaultOptions is a set of values to be selected by default.
+func promptUserMulti(msg string, options []string, defaultOptions []string, colorization colors.Colorization) []string {
+	prompt := "\b" + colorization.Colorize(colors.SpecPrompt+msg+colors.Reset)
+	surveycore.DisableColor = true
+	surveyIcons := survey.WithIcons(func(icons *survey.IconSet) {
+		icons.Question = survey.Icon{}
+		icons.SelectFocus = survey.Icon{Text: colorization.Colorize(colors.BrightGreen + ">" + colors.Reset)}
```

Comment:
I don't know of the top of my head, is the `>` something we usually colorize?

Reply:
yes
https://github.com/pulumi/pulumi/blob/42efc3c00fa65f17d37efb87b5fc856b14c49187/pkg/cmd/pulumi/util.go#L1168-L1169